doc/userguide/rules/dns-keywords.rst

```diff
@@ -46,6 +48,47 @@ Match on DNS requests where the **opcode** is NOT 0::
 
   dns.opcode:!0;
 
+Match on DNS requests where the **opcode** is between 7 and 15, exclusively:
+
+  dns.opcode:7-15;
+
+Match on DNS requests where the **opcode** is not between 7 and 15:
+
+  dns.opcode:!7-15;
+
+dns.rcode
+----------
```

Comment:
nit: make sure that the number of `-` characters matches that in the text above. In this case, there's one extra `-`